src/transformers/modeling_gpt2.py

```diff
@@ -118,7 +118,7 @@ def load_tf_weights_in_gpt2(model, config, gpt2_checkpoint_path):
 
 
 class Attention(nn.Module):
-    def __init__(self, nx, n_ctx, config, scale=False):
+    def __init__(self, nx, n_ctx, config, scale=False, is_cross_attention=False):
```

Comment:
need to add a flag here because layers will be different depending on whether it's cross attention or normal attention